plutus-use-cases/test/Spec/GameStateMachine.hs

```diff
@@ -318,27 +316,27 @@ tests :: TestTree
 tests =
     testGroup "game state machine with secret arguments tests"
     [ checkPredicateOptions options "run a successful game trace"
-        (walletFundsChange w2 (Ada.toValue Ledger.minAdaTxOut <> Ada.adaValueOf 3 <> guessTokenVal)
+        (walletFundsChange w2 (Ada.adaValueOf 3 <> guessTokenVal)
         .&&. valueAtAddress (Scripts.validatorAddress $ G.typedValidator gameParam) (Ada.adaValueOf 5 ==)
-        .&&. walletFundsChange w1 (Ada.toValue (-Ledger.minAdaTxOut) <> Ada.adaValueOf (-8)))
+        .&&. walletFundsChange w1 (Ada.adaValueOf (-8)))
         successTrace
 
     , checkPredicateOptions options "run a 2nd successful game trace"
         (walletFundsChange w2 (Ada.adaValueOf 3)
         .&&. valueAtAddress (Scripts.validatorAddress $ G.typedValidator gameParam) (Ada.adaValueOf 0 ==)
-        .&&. walletFundsChange w1 (Ada.toValue (-Ledger.minAdaTxOut) <> Ada.adaValueOf (-8))
-        .&&. walletFundsChange w3 (Ada.toValue Ledger.minAdaTxOut <> Ada.adaValueOf 5 <> guessTokenVal))
+        .&&. walletFundsChange w1 (Ada.adaValueOf (-8))
+        .&&. walletFundsChange w3 (Ada.adaValueOf 5 <> guessTokenVal))
         successTrace2
 
     , checkPredicateOptions options "run a successful game trace where we try to leave 1 Ada in the script address"
-        (walletFundsChange w1 (Ada.toValue (-Ledger.minAdaTxOut) <> guessTokenVal)
-        .&&. valueAtAddress (Scripts.validatorAddress $ G.typedValidator gameParam) (Ada.toValue Ledger.minAdaTxOut ==))
+        (walletFundsChange w1 (guessTokenVal)
+        .&&. valueAtAddress (Scripts.validatorAddress $ G.typedValidator gameParam) (Ada.toValue 1344798 ==))
```

Comment:
What's this magic number?

Reply:
It's a calculated cost by `evaluateMinLovelaceOutput` that is necessary for a `txOut` with 1 token.

As long as we don't use a test framework (`ContractModel`) for this test where we can change wallets balances we have to check values.

We can introduce a variable with a name and a comment that will cover that.

Reply:
Or we change the testcase to try to leave 2 Ada, so this can just be `2_000_000`?